_data/other.yml

```diff
@@ -30,6 +30,13 @@ websites:
       sms: Yes
       software: Yes
 
+    - name: Any.do
+      url: https://web.any.do/
+      img: anydo.png
+      tfa: No
+      twitter: AnyDO
+      facebook: Any.DO
```

Comment:
Yes. I should have clarified by initial comment. Anyway, thanks for correcting the case. `looks-good`